packages/fcl/tsconfig-web.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
I've just removed the `tsconfig-web` & `tsconfig-react-native` configs.  I don't think types were working properly for react native anyways & we are going to split into separate packages so this is unnecessary